context/app/static/js/pages/Dataset/hooks.ts

```diff
@@ -136,9 +136,9 @@ function getProcessedDatasetSection({
   const shouldDisplaySection = {
     summary: true,
     visualization: visualization || Boolean(conf && 'data' in conf && conf?.data),
-    files: Boolean(files?.length !== 0),
+    files: Boolean(files?.length),
     analysis: Boolean(metadata?.dag_provenance_list),
-    attribution: creation_action !== 'Central Process' && Boolean(contributors?.length !== 0),
+    attribution: creation_action !== 'Central Process' && !!contributors?.length,
```

Comment:
Why `!!`?

Reply:
I wanted to match the syntax of the other `contributors` check for consistency's sake - is the worry that this might be unnecessarily confusing?

Reply:
I think it's somewhat more confusing than `Boolean(contributors?.length)`.